### PR TITLE
[DOC] Fix rename processor 'otel_trace' to 'otel_traces'

### LINKED ISSUE #4935: [DOC] rename processor 'otel_trace' to 'otel_traces'
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.

With the release of Data-Prepper version 2.2.0 the 'otel_trace_raw' processor has been [renamed](https://github.com/opensearch-project/data-prepper/tree/2.4.0/data-prepper-plugins/otel-trace-raw-processor) to 'otel_traces'. The [OpenSearch documentation](https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/otel-trace-raw/) states the processor as 'otel_trace'. So simply the "s" is missing, which leads to an exception when using it.

https://github.com/opensearch-project/data-prepper/tree/2.4.0/data-prepper-plugins/otel-trace-raw-processor
https://github.com/opensearch-project/data-prepper/releases/tag/2.2.0
https://opensearch.org/docs/latest/data-prepper/pipelines/configuration/processors/otel-trace-raw/
 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

I've also tried to use the processor 'otel_trace' (without s) and data-prepper version 2.3.0 and it throws the exception:
```
org.opensearch.dataprepper.model.plugin.NoPluginFoundException: Unable to find a plugin named 'otel_trace'. Please ensure that plugin is annotated with appropriate values.
```

### PR DESCRIPTION
**Pull request by samipak458**
### Description
Fix 'otel_trace_raw' processor [renamed](https://github.com/opensearch-project/data-prepper/tree/2.4.0/data-prepper-plugins/otel-trace-raw-processor) to 'otel_traces'. Simply add the missing "s" that causing an exception when using it.

### Issues Resolved
Fixes #4935

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by samipak458** [comment]
@Naarcha-AWS kindly review it.

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ The [OpenTelemetry source]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/c
...
-* *otel_trace_group* - The *otel_trace_group* processor fills in the missing trace-group-related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/298e7931aa3b26130048ac3bde260e066857df54/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend.
+* *otel_traces_raw* - The *otel_traces_raw* processor receives a collection of [span](https://github.com/opensearch-project/data-prepper/blob/fa65e9efb3f8d6a404a1ab1875f21ce85e5c5a6d/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records from [*otel-trace-source*]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/otel-trace/), and performs stateful processing, extraction, and completion of trace-group-related fields.
```
Comment (Naarcha-AWS) [inline _data-prepper/common-use-cases/trace-analytics.md:41]:
```suggestion
* *otel_traces_raw* - The *otel_traces_raw* processor receives a collection of [span](https://github.com/opensearch-project/data-prepper/blob/fa65e9efb3f8d6a404a1ab1875f21ce85e5c5a6d/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records from [*otel_traces_source*]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/otel-trace/), and performs stateful processing, extraction, and completion of trace-group-related fields.
```

File: _data-prepper/managing-data-prepper/peer-forwarder.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ parent: Managing Data Prepper
 
 # Peer forwarder
 
-Peer forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. This HTTP service uses a hash-ring approach to aggregate events and determine which Data Prepper node it should handle on a given trace before rerouting it to that node. Currently, peer forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_trace_raw` [processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
+Peer forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. This HTTP service uses a hash-ring approach to aggregate events and determine which Data Prepper node it should handle on a given trace before rerouting it to that node. Currently, peer forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_traces_raw` [processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/peer-forwarder.md:10]:
```suggestion
Peer Forwarder is an HTTP service that performs peer forwarding of an `event` between Data Prepper nodes for aggregation. This HTTP service uses a hash-ring approach to aggregate events and determine which Data Prepper node it should handle on a given trace before rerouting it to that node. Currently, peer forwarder is supported by the `aggregate`, `service_map_stateful`, and `otel_traces_raw` [processors]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
```

File: _data-prepper/managing-data-prepper/peer-forwarder.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ parent: Managing Data Prepper
 
 # Peer forwarder
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/peer-forwarder.md:8]:
```suggestion
# Peer Forwarder
```

File: _data-prepper/managing-data-prepper/peer-forwarder.md [hunk 4]
```diff
@@ -7,9 +7,9 @@ parent: Managing Data Prepper
...
 
-Peer forwarder groups events based on the identification keys provided by the supported processors. For `service_map_stateful` and `otel_trace_raw`, the identification key is `traceId` by default and cannot be configured. The `aggregate` processor is configured using the `identification_keys` configuration option. From here, you can specify which keys to use for peer forwarder. See [Aggregate Processor page](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) for more information about identification keys.
+Peer forwarder groups events based on the identification keys provided by the supported processors. For `service_map_stateful` and `otel_traces_raw`, the identification key is `traceId` by default and cannot be configured. The `aggregate` processor is configured using the `identification_keys` configuration option. From here, you can specify which keys to use for peer forwarder. See [Aggregate Processor page](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) for more information about identification keys.
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/peer-forwarder.md:12]:
```suggestion
Peer Forwarder groups events based on the identification keys provided by the supported processors. For `service_map_stateful` and `otel_traces_raw`, the identification key is `traceId` by default and cannot be configured. The `aggregate` processor is configured using the `identification_keys` configuration option. From here, you can specify which keys to use for peer forwarder. See the [Aggregate Processor page](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) for more information about identification keys.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
A few small changes.

**Comment by samipak458** [comment]
All changes done as per requested. Kindly review it again @Naarcha-AWS

File: _data-prepper/managing-data-prepper/peer-forwarder.md (hunk 4 above)
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/peer-forwarder.md:12]:
```suggestion
Peer Forwarder groups events based on the identification keys provided by the supported processors. For `service_map_stateful` and `otel_traces_raw`, the identification key is `traceId` by default and cannot be configured. The `aggregate` processor is configured using the `identification_keys` configuration option. From here, you can specify which keys to use for Peer Forwarder. See [Aggregate Processor page](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) for more information about identification keys.
```

**Review by Naarcha-AWS — APPROVED** [review]
One last tweak but otherwise LGTM.

**Pull request merged**